src/timer.rs

```diff
@@ -0,0 +1,60 @@
+use std::{thread, time};
+use bus::Bus;
+use std::io::{self, Write};
+
+const SAVE: &str = "\x1b7";
```

Comment:
We probably need something like termcap to figure out if we're able to do this at all. Otherwise we might just wreck the user's console on cmd.exe or similar.